ipalib/install/certmonger.py

```diff
@@ -595,7 +595,7 @@ def modify_ca_helper(ca_name, helper):
         old_helper = ca_iface.Get('org.fedorahosted.certmonger.ca',
                                   'external-helper')
         ca_iface.Set('org.fedorahosted.certmonger.ca',
-                     'external-helper', helper)
+                     'external-helper', helper, timeout=120)
```

Comment:
It would be nice to have a code comment here, explaining briefly why we are setting the timeout to this value.

Reply:
Done.